app/models/provider_suggestion.rb

```diff
@@ -0,0 +1,7 @@
+class ProviderSuggestion < Base
```

Comment:
Why a new model?

Reply:
I've added this separation in the backend too because I believe these are distinct domains. While a provider is persistent, a suggestion is transient and need only communicate a small amount of information, not only does does this make things easier to work with, it'll also be alot faster.

Reply:
The spec allows returning subsets of fields if that's what you are after https://jsonapi.org/format/#fetching-sparse-fieldsets

Reply:
I did consider this at one point but I feel like this implementation has better extensibility, we may for example want to add ordering data to a suggestion that wouldn't be needed as part of the provider model or some other such thing. If we're tied to the provider model we have to end up making the model even bigger than it already is.